src/dbus/mutter_display_config.rs

```diff
@@ -45,6 +82,137 @@ pub struct LogicalMonitor {
 
 #[dbus_interface(name = "org.gnome.Mutter.DisplayConfig")]
 impl DisplayConfig {
+    async fn get_resources(
```

Comment:
Is this method actually needed? The panel shows up as is for me, and I cannot actually find any calls to  `GetResources` in gnome-control-center: https://gitlab.gnome.org/search?search=GetResources&nav_source=navbar&project_id=660&group_id=8&search_code=true&repository_ref=main

Reply:
The thing that calls it is [gnome-rr](https://github.com/GNOME/gnome-desktop/blob/cf8481db03400a80188b778cc230cd0a2d7d5347/libgnome-desktop/gnome-rr/gnome-rr-screen.c#L507) but I forgot where I encountered usage of that API… ah. Not for Settings, but [it's required for gsd-color](https://github.com/NixOS/nixpkgs/issues/123772#issuecomment-845005628). I'd like to get Night Light working, at least via an external client that would read g-s-d's temperature output and do `wlr-gamma-control-unstable-v1` based on it, but for that `gsd-color` must actually work

Reply:
Maybe in a separate PR then?